### ISSUE TITLE
'Download Max Size Preview' not working

### ISSUE THREAD

**Issue by Yaruze66** [zixaphir/Stable-Diffusion-Webui-Civitai-Helper#45]
## Have you read document?
Yes.

## Have you checked console log window's msg?
Yes.

## Describe Issue
I have "Download Max Size Preview" checked ("ch_max_size_preview": true), but the previews are still downloaded compressed from the main page at a resolution of 450x675 instead of downloading the original uncompressed ones at a resolution of 1024x1536.

## Console log's msg or screenshot for function issue
```
Civitai Helper: Getting model info by: https://civitai.com/models/134215/meichilightmix
Civitai Helper: Run get_model_id_from_url
Civitai Helper: Request model info from civitai: 134215
Civitai Helper: Get subfolder for: D:\stable-diffusion-webui\models\Stable-diffusion
Civitai Helper: Got following info for downloading:
   model_name: 'MeichiLight_Mix'
   model_type: 'ckp'
   version_strs: ['MeichiLight_Mix V2.0_224422', 'v1.0_147809']
   subfolders: ['/']
   previews: {'MeichiLight_Mix V2.0_224422': [{'url': 'https://image.civitai.com/xG1nkqKTMzGDvpLrqFT7WA/17c1931e-9658-4f03-adf3-b966d32c1a96/width: 450/3608386.jpeg', 'nsfw': 'Mature', 'width': 1024, 'height': 1536, 'hash': 'U7G*Wr000e-AX,00tk-o00~Ux^%2NIIq4o~U', 'type': 'image', 'metadata': {'hash': 'U7G*Wr000e-AX,00tk-o00~Ux^%2NIIq4o~U', 'size': 2325103, 'width': 1024, 'height': 1536}, 'meta': {'Size': '512x768', 'seed': 4294686890, 'Model': 'Only meichi real 2 oda plus.fp16', 'steps': 20, 'hashes': {'model': '57d2ddc49c'}, 'prompt': 'a female elf with a sexy body figure, hunting, medieval theme, pointy ears, ((MILF)), seducing smile, seducing looking, \n(((masterpiece))), ((best quality)), ((intricate detailed)), a sexy woman with elf attributes cloths, (huge breast), bracers, pauldrons, perspective, highly detailed, illustration, 1 girl, perfect hands, detailed fingers, beautiful detailed eyes, long hair, blonde hair (fantasy:1.2), armor, detailed background, at the castle, bedroom, royal, painting on the wall, fierce look, looking at the viewer, from the front,', 'Version': 'v1.3.0', 'sampler': 'DPM++ SDE Karras', 'cfgScale': 7, 'clipSkip': 2, 'resources': [{'hash': '57d2ddc49c', 'name': 'Only meichi real 2 oda plus.fp16', 'type': 'model'}], 'Model hash': '57d2ddc49c', 'Hires steps': '15', 'Hires upscale': '2', 'Hires upscaler': 'R-ESRGAN 4x+ Anime6B', 'negativePrompt': '(worst quality, low quality:2), monochrome, zombie,overexposure, watermark,text, ((bad anatomy, bad proportions)), bad composition, too many fingers,fused fingers,bad arm,distorted arm,extra arms,fused arms,extra legs,missing leg,disembodied leg,extra nipples, detached arm, liquid hand,inverted hand,disembodied limb, small breasts, loli, oversized head,extra body,completely nude, extra navel,easynegative,(hair between eyes),sketch, duplicate, ugly, huge eyes, text, logo, worst face, (bad and mutated hands:1.3), (blurry:2.0), horror, geometry, bad_prompt, ((anime_badhandv4, bad-hands-5)), (bad hands), (broken hands, broken fingers, deformed hands, deformed fingers, missing fingers, missing hands, extra fingers, extra hands:1.5), multiple limbs, (interlocked fingers:1.2), Ugly Fingers, (extra digit and hands and fingers and legs and arms:1.4), ((2girl)), (deformed fingers:1.2), (long fingers:1.2),(bad-artist-anime), bad-artist, bad hand, extra legs ,(ng_deepnegative_v1_75t), ((deformed body)), ((awkward pose, bad pose)), over saturated,outdoors, ( white skin:1.4),', 'ADetailer model': 'face_yolov8n.pt', 'ADetailer prompt': '" \\"beautiful detailed green eyes,detailed lips,eye shadow,detailed face,looking at viewer,face blush,pout\\""', 'Noise multiplier': '0', 'ADetailer version': '23.7.1', 'Denoising strength': '0.5', 'ADetailer mask blur': '4', 'ADetailer confidence': '0.3', 'ADetailer dilate/erode': '32', 'ADetailer inpaint padding': '0', 'ADetailer negative prompt': '"easynegative', 'ADetailer denoising strength': '0.4', 'ADetailer inpaint only masked': 'True'}}...
Civitai Helper: Searching local model by version id
Civitai Helper: folder: D:\stable-diffusion-webui\models\Stable-diffusion\
Civitai Helper: version_id: 224422
Civitai Helper: Start downloading from: https://civitai.com/api/download/models/224422
Civitai Helper: Target file path: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.safetensors
Civitai Helper: File size: 5234115818 (4.87G)
Civitai Helper: Downloading to temp file: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.safetensors.downloading
100%|████████████████████████████████████████████████████████████████████████████| 4.87G/4.87G [08:19<00:00, 10.5MiB/s]
Civitai Helper: File Downloaded to: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.safetensors
Civitai Helper: Request version info from civitai
Civitai Helper: Fetching Parent Model Information
Civitai Helper: Request model info from civitai: 134215
Civitai Helper: Write model civitai info to file: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.civitai.info
Civitai Helper: Write model webui info to file: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.json
Civitai Helper: Downloading model image.
Civitai Helper: Checking preview image for model: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.safetensors
Civitai Helper: Start downloading from: https://image.civitai.com/xG1nkqKTMzGDvpLrqFT7WA/17c1931e-9658-4f03-adf3-b966d32c1a96/width=450/3608386.jpeg
Civitai Helper: Target file path: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.preview.png
Civitai Helper: File size: 71766 (70.08K)
Civitai Helper: Downloading to temp file: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.preview.png.downloading
100%|█████████████████████████████████████████████████████████████████████████████| 70.1k/70.1k [00:00<00:00, 946kiB/s]
Civitai Helper: File Downloaded to: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.preview.png
Civitai Helper: Done. Downloaded to: D:\stable-diffusion-webui\models\Stable-diffusion\meichilightMix_meichilightMixV20.safetensors

```
Caution: Soft NSFW!
[The compressed image I'm getting](https://image.civitai.com/xG1nkqKTMzGDvpLrqFT7WA/17c1931e-9658-4f03-adf3-b966d32c1a96/width=450/tmphtipb_e5.jpeg)

[Full-size image](https://image.civitai.com/xG1nkqKTMzGDvpLrqFT7WA/17c1931e-9658-4f03-adf3-b966d32c1a96/width=1024/tmphtipb_e5.jpeg)

**Comment by zixaphir** [comment]
This was an oversight on my part. I'll try to get it fixed by the end of the day

**Comment by zixaphir** [comment]
This should be fixed in 1.8.1
